conf/app.routes

```diff
@@ -40,9 +40,9 @@ GET        /dependencies/:name/:version                 uk.gov.hmrc.cataloguefro
 GET        /dependencies/:name/:version/graphs          uk.gov.hmrc.cataloguefrontend.DependenciesController.graphs(name, version, scope: String ?= "compile")
 
 GET        /dependencyexplorer                          uk.gov.hmrc.cataloguefrontend.DependencyExplorerController.landing
-GET        /dependencyexplorer/results                  uk.gov.hmrc.cataloguefrontend.DependencyExplorerController.search
+GET        /dependencyexplorer/results                  uk.gov.hmrc.cataloguefrontend.DependencyExplorerController.search(team: String ?= "", flag: String ?= "", `scope[]`: Seq[String] ?= Seq.empty, group: String ?= "", artefact: String ?= "", versionRange: String ?= "", asCsv: Boolean ?= false)
 
-GET        /bobbyrulestrend                             uk.gov.hmrc.cataloguefrontend.BobbyRulesTrendController.display
+GET        /bobbyrulestrend                             uk.gov.hmrc.cataloguefrontend.BobbyRulesTrendController.display(`rules[]`: Seq[String] ?= Seq.empty, from: java.time.LocalDate ?= java.time.LocalDate.now().minusYears(2), to: java.time.LocalDate ?= java.time.LocalDate.now())
```

Comment:
this allows encoding urls by default when using reverse routes, in order to avoid this isse:
```
Bad value “/bobbyrules#uk.gov.hmrc:reactivemongo-test:[4.0.0,4.4.0]” for attribute “href” on element “a”: Illegal character in fragment: “[” is not allowed.
```